src/app/services/cities.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import {HttpClient} from "@angular/common/http";
+import {Observable} from "rxjs";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CitiesService {
+
+  constructor(private httpClient: HttpClient) {}
+
+  getCities(nameCity: string): Observable<any> {
+    return this.httpClient.get(`https://geo.api.gouv.fr/communes?nom=${nameCity}`);
```

Comment:
`this.httpClient.get<string>('https://geo.api.gouv.fr/communes?nom=${nameCity}');`